### PR TITLE
Control spec update- Kunernetes, NotifiationHub

### PR DESCRIPTION
**Pull request by Aboli-msft**

### CODE REVIEW

File: Control coverage/Feature/KubernetesService.md [hunk 1]
```diff
@@ -3,4 +3,8 @@
...
+|<b>ControlId:</b><br>Azure_KubernetesService_Audit_Enable_Monitoring<br><b>DisplayName:</b><br>Monitoring must be enabled for Azure Kubernetes Service.<br><b>Description: </b><br>Monitoring must be enabled for Azure Kubernetes Service. | <b> ARM API to list Container Services at subscription level: </b> <br> /subscriptions/{subscriptionId}/providers/Microsoft.ContainerService/managedClusters?api-version=2020-09-01<br><b>Properties:</b><br> properties.addonProfiles.omsagent.enabled  | <b>Passed: </b><br> Kubernetes resource have LA configured and monitoring is enabled. <br><b>Failed: </b><br>Kubernetes resource do not have LA configured and monitoring  is disabled.|
```
Comment (TarunKrShukla) [inline Control coverage/Feature/KubernetesService.md:7]: Plz dont user '.' fullstop for displayName or description.

Comment (TarunKrShukla) [inline Control coverage/Feature/KubernetesService.md:7]: ARM API to list Container Services at subscription level , looks this is copy paste error in all controls
This should be changed to Kubernetes Services

File: Control coverage/Feature/KubernetesService.md [hunk 2]
```diff
@@ -3,4 +3,8 @@
...
+|<b>ControlId:</b><br>Azure_KubernetesService_NetSec_Dont_Open_Management_Ports<br><b>DisplayName:</b><br>Do not leave management ports open on Kubernetes nodes.<br><b>Description: </b><br>Do not leave management ports open on Kubernetes nodes unless required. | <b> ARM API to list Container Services at subscription level: </b> <br> /subscriptions/{subscriptionId}/providers/Microsoft.ContainerService/managedClusters?api-version=2020-09-01<br> | <b>Passed: </b><br> NSG is configured and no inbound port is open or if restricted ports not found.<br><b>Failed: </b><br>NSG is not configured or if restricted ports found.<br><b>NotApplicable</b>Control is not applicable for ERvNET connected AKS VM/VMSS:|
```
Comment (TarunKrShukla) [inline Control coverage/Feature/KubernetesService.md:9]: No properties are mentioned for this control, Also there are typos in Control speclet column.

File: Control coverage/Feature/KubernetesService.md [hunk 3]
```diff
@@ -3,4 +3,8 @@
...
+|<b>ControlId:</b><br>Azure_KubernetesService_Deploy_Use_Latest_Version<br><b>DisplayName:</b><br>Kubernetes Services should be upgraded to a non-vulnerable Kubernetes version.<br><b>Description: </b><br>The latest version of Kubernetes should be used. | <b> ARM API to list Container Services at subscription level: </b> <br> /subscriptions/{subscriptionId}/providers/Microsoft.ContainerService/managedClusters?api-version=2020-09-01<br><b>Properties:</b><br>properties.kubernetesVersion <br><b>Azure built-in policy:</b><br>Kubernetes Services should be upgraded to a non-vulnerable Kubernetes version | <b>Passed: </b><br> AKS is running on one of the defined version <br><b>Failed: </b><br>AKS is not running on the required kubernetes version..|
```
Comment (TarunKrShukla) [inline Control coverage/Feature/KubernetesService.md:10]: Two full stops in control spec-let

File: Control coverage/Feature/NotificationHub.md [hunk 4]
```diff
@@ -0,0 +1,5 @@
+## Kubernetes Service
+
+| ControlId | Dependent Azure API(s) and Properties | Control spec-let |
+|-----------|-------------------------------------|------------------|
```
Comment (TarunKrShukla) [inline Control coverage/Feature/NotificationHub.md:4]: Plz don't use fullstop '.' in display name and description.

File: Control coverage/Feature/NotificationHub.md [hunk 5]
```diff
@@ -0,0 +1,5 @@
...
+| <b>ControlId:</b><br>Azure_NotificationHub_AuthZ_Dont_Use_Manage_Access_Permission	<br><b>DisplayName:</b><br>Access policies on Notification Hub must not have Manage access permissions.<br><b>Description: </b><br> Access policies on Notification Hub must not have Manage access permissions. | <b> ARM API to get Authorization rules: </b> <br> /subscriptions/{subsriptionId}/resourceGroups/{REsourcegroupName}/providers/Microsoft.NotificationHubs/namespaces/{NamespaceName}/notificationHubs/{NotificationHubName}/AuthorizationRules?api-version=2016-03-01 <br><b>Properties:</b><br> properties.rights<br>name<br> | <b>Passed: </b><br> No authorization rule Found with manage permission. <br><b>Verify: </b><br>Authorization rules found with manage permission.|
```
Comment (TarunKrShukla) [inline Control coverage/Feature/NotificationHub.md:5]: If there are multiple properties for a give API, use comma ',' to separate them instead of newline.

**Review by TarunKrShukla — CHANGES_REQUESTED** [review]
Changes needed as per comments.

**Comment by VishalHaibatpure** [comment]
@Aboli-msft: Do we need to drop this PR as we have moved to new format?

**Comment by Aboli-msft** [comment]
With new format this PR is not needed

**Pull request closed without merging**